package.json

```diff
@@ -7,7 +7,9 @@
     "node": ">=12.0.0"
   },
   "scripts": {
-    "test": "mocha -r ts-node/register 'tests/**/*.scrypttest.*'  --reporter spec --timeout 600000 -g '' ",
+    "test": "npm run test:withoutHeavy",
+    "test:withoutHeavy": "mocha -r ts-node/register 'tests/**/*.scrypttest.*'  --reporter spec --timeout 10000 -i -g 'Heavy'",

```

Comment:
👍 nice

-f may be better than -g:
See Running matching tests in https://blog.logrocket.com/a-quick-and-complete-guide-to-mocha-testing-d0e0ea09f09d/